### PR TITLE
DNM changes to print, removing latex printing

### PR DESCRIPTION
**Pull request by evykassirer**
wait for athena to merge latex

### CODE REVIEW

File: lib/expression/step-solver/simplifyExpression.js [hunk 1]
```diff
@@ -128,25 +131,24 @@ function step(node) {
 // Returns the new steps array.
 function addStep(steps, nodeStatus, debug) {
   let explanation = nodeStatus.changeType;
```
Comment (evykassirer) [inline lib/expression/step-solver/simplifyExpression.js]: does this line help make things more clear? I prefer `'explanation': nodeStatus.changeType,` below

File: lib/expression/step-solver/solveEquation.js [hunk 2]
```diff
@@ -140,12 +140,11 @@ function addSimplificationSteps(steps, equation, firstTime, debug=false) {
 // adds a step to the given steps based on the given equationStatus
 function addStep(steps, equationStatus, debug) {
   let explanation = equationStatus.changeType;
-  let asciimath = equationStatus.equation.prettyPrint();
-  let latex = equationStatus.equation.prettyPrint(true);
+  let asciimath = equationStatus.equation.print();
```
Comment (evykassirer) [inline lib/expression/step-solver/solveEquation.js]: same here

File: lib/expression/step-solver/simplifyExpression.js [hunk 3]
```diff
@@ -56,16 +57,19 @@ function stepThrough(node, debug=false, firstTime=true) {
...
   // Now, step through the math expression until nothing changes
   nodeStatus = step(node);
   while (nodeStatus.hasChanged()) {
     steps = addStep(steps, nodeStatus, debug);
-    nodeStatus.resetchangeGroups();
-    nodeStatus = step(nodeStatus.newNode);
+    nodeStatus.newNode = clone(nodeStatus.newNode);
+    nodeStatus.resetChangeGroups();
+    // clone so that the next step's oldNode doesn't modify this step's newNode
+    // TODO: when clone() is accessibile everywhere, move this to resetChangeGroups
```
Comment (aelnaiem) [inline lib/expression/step-solver/simplifyExpression.js]: Can this happen with my clone change?

File: lib/expression/step-solver/simplifyExpression.js (hunk 1 above)
Comment (aelnaiem) [inline lib/expression/step-solver/simplifyExpression.js]: Nah you can change it to your preference np

**Comment by aelnaiem** [comment]
Looks good!

**Comment by evykassirer** [comment]
gonna wait to merge the athena change before I merge this

**Comment by aelnaiem** [comment]
👍

**Pull request merged**